CMakeLists.txt.user

```diff
@@ -0,0 +1,876 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be merged into the git repository, can you please remove it?
It was created by CMake on your machine and is machine specific. I assume that this file was created by QtCreator? I see two solutions:
- Tell QtCreator to create these files somewhere else
- Add an entry to the .gitignore file so the file is ignored in the future.

Reply:
Deleted.